build-tools/src/main/java/uk/ac/ox/ndph/mts/json_schema_generator/model/InitTrialConfig.java

```diff
@@ -0,0 +1,19 @@
+package uk.ac.ox.ndph.mts.json_schema_generator.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+import uk.ac.ox.ndph.mts.init_service.model.Trial;
+
+@Component
+@Configuration
+public class InitTrialConfig {
```

Comment:
I think you should add a java doc here to explain what this class is for and why an internal Mts one. Also valid for the other classes.